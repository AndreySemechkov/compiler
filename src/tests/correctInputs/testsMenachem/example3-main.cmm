extern integer foo2();
extern integer foo(c: integer);

main
var a:integer;
    b: real;
{
	a =2;
    b = (real) call foo(a );
	write(b);
}
