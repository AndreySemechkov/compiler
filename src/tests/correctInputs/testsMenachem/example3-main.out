3.000000 Reached Halt.
